user-docs/documentation/docs/how-to-guides/4-configure-ci-cd.md

```diff
@@ -6,17 +6,35 @@ sidebar_position: 4
 
 ## Choose your CI/CD provider
 
-_This documentation is still a work in progress. If you wish to contribute, please [open an issue on Swarmion](https://github.com/swarmion/swarmion/issues)._
+By default, Swarmion uses Github Actions to deploy. However, we have already used GitlabCI and CircleCI to deploy Swarmion projects. This guide will focus on Github Actions.
 
-By default, Swarmion uses Github Actions to deploy. However, we have already used GitlabCI and CircleCI to deploy Swarmion projects.
+## Setup the CD authentication
 
-## Authorize your CD to access your AWS environments
+In order to be able to deploy resources on AWS, your CI needs **authentication** and **permissions**. There are two main ways to provide authentication:
 
-In order to restrict the abilities of the ci user on the testing and production environment, you need to create one or many policies to give that user.
+- create an IAM user on the target AWS account, retrieve its credentials
+- create an IAM role that can be assumed by a trusted party
 
-If you use Github Actions, we can recommend to follow [these guidelines to setup OIDC](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services), to authorize your pipeline.
+We strongly recommend the second option, as it will remove the risk of losing long-lived credentials. The Swarmion CI/CD from all starters, uses this method by default.
 
-This will remove the need to store long lived credentials, and thus the need to specify a policy (we can grant admin access). If you do so, you can skip the following sections.
+### With OpenID Connect (OIDC)
+
+If you use Github Actions, we can recommend to follow [these guidelines](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services) to setup OIDC for your pipeline.
+
+If you are using the Swarmion generated CI, once you have created the role, retrieve its ARN and place it in a Github Actions secret named `AWS_ROLE_ARN_PRODUCTION`.
+
+### With an IAM user
```

Comment:
We could separate the two parts with [tabs](https://docusaurus.io/docs/markdown-features/tabs), so it will be clearer that the restriction is not recommend for OIDC